### DISCUSSION TITLE
[Feature Request] Sort option for Books: Autor, Title, Year

### DISCUSSION THREAD

**Discussion by Toschko** [PaulWoitaschek/Voice#1543] [Ideas]
It would be nice to have a the possibility to sort book by   author, title or year. I have many books from the same author and it is difficult to hear them in the correct order.
The order should be stored. 
When You sort by Title and than by author, the order from the titles should n't change,. So the books are sorted by author, title

**Comment by PaulWoitaschek** [comment]
What about searching for the author instead? Does that already cover your use case?

**Reply by Toschko** [reply]
Sure, thats possible, but not so convenient and also there should be some sorting in the list. I don't like disorder on my book shelf ;-)

**Comment by ghost** [comment]
Especially for book series custom sorting or by release date would be great. At the moment custom sorting can only be achieved by changing the name

**Comment by xcy7e** [comment]
#### Custom sort workaround
Before v6.3.0 one could workaround custom sorting by moving books to 'current' (top section) and then back to 'new' (middle section) as it would appear and stay on top then.

Obviously it was impossible to sort anything in between, but - at least - could keep the favourite next items on top.

**Comment by mwanderman** [comment]
I'd also like an option to sort by length. For example, if I know I'll have seven hours to listen to an audiobook over the next couple days, I could easily see what books I could start and finish in that time.
